src/Makefile

```diff
@@ -49,15 +41,15 @@ check-physlr-filter-bxmx-second: all
 	./physlr-filter-bxmx data/f1.tiny.indexlr.n2-10.physlr.tsv -o f1.tiny.indexlr.n2-10.c2-x.physlr.tsv -n2 -N10
 	./data/checkMx.sh f1.tiny.indexlr.n2-10.c2-x.physlr.tsv data/f1.tiny.indexlr.n2-10.c2-x.physlr.tsv.good
 
-check-physlr-index-stLFR: all
-	./physlr-indexlr -t16 -k8 -w1 data/stLFR.tiny.fq | diff -q - data/stLFR.tiny.physlr.tsv.good
+#check-physlr-index-stLFR: all
```

Comment:
Should we just delete? Since they're commented out anyway?

Reply:
I just did not want to deal with them right now... but ideally would be good to have them... so I think we may let them be there as a reminder? I'm okay either way and if it's not good practice I can remove them.

Reply:
Yeah whichever you prefer! I guess that all that would need to be done is re-run `indexlr` on the `data/stLFR.tiny.fq` to bring the test back?